queries.sql

```diff
@@ -1,3 +1,28 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * from animals
+WHERE name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals
+WHERE 
+    date_of_birth BETWEEN '2016-01-01'
+and  '2019-01-01';
+
```

Comment:
- [ ] Here, you should list the name of all animals that are neutered and have less than 3 escape attempts.